docs/git/index.md

```diff
@@ -0,0 +1,220 @@
+---
+id: index
+title: Introduction
+sidebar_label: Introduction
+---
+
+Most software engineers use a tool called Git to organise their code, and collaborate with other people.
+
+Git can be used for many things, and we will see lots of them throughout the course.
+
+Each week, you're going to use Git to get a copy of your homework exercises, to submit your solutions, and to get feedback on your solutions. Today, you're going to learn how to do these things.
+
+## Learning Objectives
+
+By the end of this class, you should be able to:
+
+* Get a copy of your homework questions onto your computer.
+  * (To _clone_ a repository).
+* Save your answers, and send them to our volunteers.
+  * (To _commit_ and _push_ changes).
+* See and respond to feedback on your homework.
+  * (To _fork_ a repository, use Github's _pull request_ workflows, and _push_ further changes).
+* Explore how a file has changed over time.
+  * (To use Git's _history log_).
+
+---
+
+## Prerequisites
+
+1. A Github account. If you don't have one already, go to https://github.com/ and sign up for one.
+2. Installed Github Desktop. If you don't have it yet, go to https://desktop.github.com/ and follow its instructions to install it.
+3. Sign in to Github Desktop with your Github account. Run Github Desktop (TODO: Instructions on how to find it), click "Sign in to GitHub.com", and go through the steps to sign in.
+4. Installed VS Code.
+
+## What problem does Git solve?
```

Comment:
From @40thieves:

Maybe we could add an exercise here? Maybe something where we can discuss other "versioning systems" like naming files `_v1`, `_v2` etc. And what the problems with those systems are.

Maybe something like: 

1. Student A has a file named `doc_v1`, which she shares with Students B and C
2. Students B and C make changes (in parallel) to their copies, naming them `doc_v2`
3. Students B and C return their copies to Student A, who then has to resolve the differences manually

Or:

1. The teacher prepares files `doc_v1`, `doc_v2` and `doc_v3`. The documents should be relatively complex
2. The students then have to find all the differences between the versions

Reply:
Sounds great, love it!